### ISSUE TITLE
Images not updated on website for W2KG209840

### ISSUE THREAD

**Issue by TBRC-Travis** [buda-base/public-digital-library#789]
@eroux I recently synced new images for Volumes 107, 108, and 109 (I2KG209969, I2KG209970, and I2KG209971) in W2KG209840.  For some reason the images aren't updating on the website despite being synced properly.  Can you take a look?

**Comment by eroux** [comment]
looking at I2KG209969 directly on https://library.bdrc.io/show/bdr:I2KG209969#open-viewer , the images seem to be working

looking at the collection on https://library.bdrc.io/show/bdr:W2KG209840#open-viewer I see

![Capture d’écran de 2022-12-16 15-28-20](https://user-images.githubusercontent.com/60868/208120247-fb496194-d0d6-4080-a99a-a688c99099a9.png)

how can I reproduce the issue?

**Comment by TBRC-Travis** [comment]
Yes the images load but it's still loading the old images.  For example, the website shows Volume 108 as having 542 images but if you look on s3 for this volume (s3://archive.tbrc.org/Works/4f/W2KG209840/images/W2KG209840-I2KG209970/) there are 603 images.

**Comment by eroux** [comment]
ah ok yes, I missed that in your initial report. I'll ask Bruno to update the pagination lists

**Comment by TBRC-Travis** [comment]
great, please let me know when they've been updated.  thanks!
